companion/src/modeledit/setup.cpp

```diff
@@ -238,9 +238,9 @@ ModulePanel::ModulePanel(QWidget * parent, ModelData & model, ModuleData & modul
   lock = false;
 
   // a change will not register if only one item was added
-  if (ui->protocol->count() == 1)
+  if (ui->protocol->count() == 1) {
```

Comment:
Aha yes, but we agree that we don't agree :)
I also don't like the single line
Otherwise I have to admit that I change my mind everyday